app/upgrades.go

```diff
@@ -0,0 +1,122 @@
+package app
+
+import (
+	"context"
+	"fmt"
+
+	storetypes "cosmossdk.io/store/types"
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+)
+
+const UpgradeName = "v10"
+
+func (app *WasmApp) RegisterUpgradeHandlers() {
+	upgradeInfo, err := app.UpgradeKeeper.ReadUpgradeInfoFromDisk()
```

Comment:
Does this mean that we no longer keep any upgrade information in the codebase? How will this work with integration tests?